index.html

```diff
@@ -26,7 +26,7 @@ <h1>You might <span class="highlighted">no</span>t need <span class="highlighted
         <p>jQuery and its cousins are great, and by all means use them if it makes it easier to develop your application.</p>
         <p>
           If you're developing a library on the other hand, please take a moment to consider if you actually need jQuery as a dependency.
-          Maybe you can include a few lines of utility code, and forgo the requirement.  If you're only targeting more modern browsers,
+          Maybe you can include a few lines of utility code, and forget the requirement.  If you're only targeting more modern browsers,
```

Comment:
[Forgo](https://www.merriam-webster.com/dictionary/forgo) is an acceptable word here